worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java

```diff
@@ -105,6 +105,55 @@ public SelectionCommands(WorldEdit we) {
         this.we = we;
     }
 
+    @Command(
+        name = "/pos",
+        desc = "Set positions"
+    )
+    @Logging(POSITION)
+    @CommandPermissions("worldedit.selection.pos")
+    public void pos(Actor actor, World world, LocalSession session,
+                    @Arg(desc = "Coordinates to set position 1 to", def = "")
+                        BlockVector3 pos1,
+                    @Arg(desc = "Coordinates to set position 2 to", def = "")
+                        BlockVector3 pos2,
+                    @Arg(desc = "Coordinates to add as extras if selection type supports", def = "", variable = true, name = "pos2")
```

Comment:
```suggestion
                    @Arg(name = "pos2", desc = "Coordinates to add as extras if selection type supports", def = "", variable = true)
```
I'd prefer to have this exceptional override in front for visibility.